### PR TITLE
Awesomebook

### PR DESCRIPTION
**Pull request by znation2010**
Project Requirement
- Create a collection that keeps a list of books (hint: you can use an array of objects for 
- Create a function to add a new book to the collection, with title and author.
-  Create a function to remove a book from the collection (hint: you can use the [array filter()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter) method).
- Display all books saved in the collection in the top part of the page.
- Make sure that when a user clicks the "Add" button:
        A new book is added to the collection.
        The new book is displayed in the page.
- Make sure that when a user clicks the "Remove" button:
        The correct book is removed from the collection.
        The correct book dissapears from the page.
- Make sure that data is preserved in the browser's memory by using [localStorage](https://github.com/microverseinc/curriculum-html-css/blob/main/local_storage.md).
- To debug your application, you can use [console.log()](https://developer.mozilla.org/en-US/docs/Web/API/Console/log). Remember to remove debugging code before pushing your code to the remote repository.
- Don't use any library or framework, use only plain JavaScript.

### CODE REVIEW

**Review by AbelG101 — COMMENTED** [review]
# Hi @znation2010 & @H-Ramalan  👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- Hey Zerihun, I noticed you've made some commits using a GitHub account that's not recognized by Microverse please make sure to only use 1 account that's recognized by Microverse when working on projects.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by znation2010** [comment]
which  GitHub account isn't recognized by Microverse is it  @znation2010 or @H-Ramalan

**Comment by znation2010** [comment]
I didn't understand the issue and whose account is responsible for error please explain to me the problem in detail

File: README.md [hunk 1]
```diff
@@ -42,192 +14,134 @@ After you're finished please remove all the comments and instructions!
...
 - [🙏 Acknowledgements](#acknowledgements)
-- [❓ FAQ (OPTIONAL)](#faq)
 - [📝 License](#license)
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
-
-> Describe your project in 1 or 2 sentences.
+# 📖 [Capstone Project : Module 1 - HTML, CSS, JS] <a name="about-project"></a>
```
Comment (ShadyShawkat) [inline README.md]: -[ ] Kindly update the readme information to better suit the project.

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @znation2010, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- All linters are passing ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ishimwezachee — APPROVED** [review]
Hi @AbelG101  and @ShadyShawkat 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- All linters are passing ✔️
- Descriptive commit messages ✔️
- Professional looking pull request ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ishimwezachee — APPROVED** [review]
Hi @AbelG101  and @ShadyShawkat 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
image

## Highlights
- All linters are passing ✔️
- Descriptive commit messages ✔️
- Professional looking pull request ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**